server/app.js

```diff
@@ -200,13 +200,16 @@ app.get('/user/:username', async (req, res, next)=>{
 	});
 
 	if(ownAccount && req?.account?.googleId){
-		const googleBrews = await GoogleActions.listGoogleBrews(req, res)
-		.catch((err)=>{
-			console.error(err);
-		});
-
-		if(googleBrews)
+		const auth = await GoogleActions.authCheck(req.account, res);
+		let googleBrews = await GoogleActions.listGoogleBrews(auth)
+			.catch((err)=>{
+				console.error(err);
+			});
+
+		if(googleBrews) {
+			googleBrews = googleBrews.map((brew)=>({ ...brew, authors: [req.account.username] }));

```

Comment:
```suggestion
```

I don't think this is the place to be adding the user to the authors. This should probably be handled on brew update or somewhere? Let's handle this later on when the stubs are more integrated.

Reply:
I did that because that's effectively what it was doing but it was doing it in the listGoogleBrews method using the same request member variable.

Reply:
Mmmmmmmmmmmmmmmk. I'll keep it for now then but let's see if we can move this into the stub logic later on.

Reply:
Oh yeah it will not exist anywhere once stubs are in place